documentation/asciidoc/computers/config_txt/conditional.adoc

```diff
@@ -96,6 +96,16 @@ Some models of Raspberry Pi, including Zero, Compute Module, and Keyboard models
 
 The `[none]` filter prevents any settings that follow from being applied to any hardware. Although there is nothing that you can't do without `[none]`, it can be a useful way to keep groups of unused settings in config.txt without having to comment out every line.
 
+=== The `[partition=N]` filter
+This `partition` filter can be be used in the bootloader config (only) to remap high partition numbers set either by the `sudo reboot N` command or direct usage of the hardware watchdog registers.
+
+Example
+[source,ini]
+----
+[partition=62]
+PARTITION=2
```

Comment:
Does this example mean that `sudo reboot 62` will effectively be the same as `sudo reboot 2` ? I wonder if it might be worth explaining why that's useful? :thinking:

Reply:
Updated example to show config.txt selecting alternate cmdline.txt. So the image in '2' can be differently according to the reboot parameter.